applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.hpp

```diff
@@ -191,10 +192,9 @@ namespace Kratos
 			ProcessInfo& rCurrentProcessInfo);
 
 	private:
-		double mdPhi_x_a, mRotInertiaY, mRotInertiaZ;
-		Vector mNX, mNY, mNZ, mRHS, mTotalDef, mTotalPos;
-		Vector mTotalNodalDeformation, mTotalNodalPosistion, mBodyForces;
-		Vector mDeformationModes, mIncrementDeformation;
+		Vector mNX, mNY, mNZ;
+		Vector mTotalNodalDeformation;
+		Vector mIncrementDeformation;
 		Matrix mLHS, mRotationMatrix;
 		bounded_matrix<double,msElementSize,msElementSize> mRotationMatrix0;

```

Comment:
why do you store the mLHS, the mRotationMatrix and the mRotationMatrix0?

if you have the quaterninon you can compute them efficiently when needed